#define goal
Command[] commands
uint32 control_frequency
---
#define result
bool complete
---
#define feedback
Sensors sensors
